package.json

```diff
@@ -72,7 +74,7 @@
   },
   "dependencies": {
     "@folio/stripes-acq-components": "~3.2.0",
-    "@folio/stripes-data-transfer-components": "^5.2.0",
+    "@folio/stripes-data-transfer-components": "^5.3.0",
```

Comment:
[SDTC 5.3.0 was published on 2022-10-21](https://github.com/folio-org/stripes-data-transfer-components/releases/tag/v5.3.0). This PR relies on the export `ACTION_TYPES`, which were [just added last week](https://github.com/folio-org/stripes-data-transfer-components/pull/235). The minimum version here must be 5.4.0.